Libraries/Text/Text/RCTTextView.m

```diff
@@ -282,6 +282,17 @@ - (NSString *)accessibilityLabel
   return _textStorage.string;
 }
 
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+- (NSString *)accessibilityValue
+{
+  NSString *superAccessibilityLabel = [super accessibilityLabel];
+  if (superAccessibilityLabel) {
+    return superAccessibilityLabel;
```

Comment:
what is this supposed to catch? Any reason not to always return _textStorage.string? 

It feels a bit odd and unexpected to be calling accessibilityLabel from the accessibilityValue implementation. But if we need to we should add a comment explaining why we're calling it on super and not self since super should generally only be called from within an override of that method itself.

Reply:
In iOS, Static Text is exposed through `accessibilityLabel`.   In the super, `RCTView`, there is code that is recursively collecting text of children and combining the results.    In macOS, Static Text is exposed through `accessibilityValue`, but it relies on the computed results of `RCTView accessibilityValue`.

However, this fix is leaving the impl of `accessibilityLabel` as-is for macOS, so VoiceOver is going to do something weird like announce the text twice.   The fix should be something like:

```objc
#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
- (NSString *)accessibilityLabel
#else
- (NSString *)accessibilityValue
#endif
{
```
at line 276.

AX props are simpler on iOS and a little more complex on macOS.   iOS only has label and value, but macOS has label, title, and value.    Below is a comparison of Accessibility Inspector views of Static Text and TextFields:
```
iOS

Text
 Label: “the text”
 Value: nil
 Traits: static text

TextInput
 Label: nil
 Value: “the text”
 Traits == nil

macOS

Text
 Label: nil
 Title: nil
 Value: “the text”
 Type: text

TextInput 
 Label: “the placeholder text”
 Title: nil
 Value: “the text”
 Type: text field
```
This fix fixes Static Text, but a separate bug I just found is that for TextInput, we are not putting the placeholder text in the `accessibilityLabel` field.   Instead we are putting in `accessibilityValue` and replacing it with the user entered text after the user types.   We should be preserving placeholder text in  `accessibilityLabel`